src/app/_components/utils/lang.ts

```diff
@@ -0,0 +1,9 @@
+import { TypeHome } from '../types/home';
```

Comment:
`element` フォルダを `i18n` にして、
このファイルを `i18n/index.ts` にしてあげた方が分かりやすいと思う
```suggestion
import { TypeHome } from '../types/home';
```